packages/pci/pci.1.0.1/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "1.2"
+maintainer:   "simonjbeaumont@gmail.com"
+authors:      "Si Beaumont"
+homepage:     "https://github.com/simonjbeaumont/ocaml-pci"
+bug-reports:  "https://github.com/simonjbeaumont/ocaml-pci/issues"
+dev-repo:     "https://github.com/simonjbeaumont/ocaml-pci.git"
+
+build: [
+  ["./configure"]
+  [make]
+]
+build-test: [
+  ["./configure" "--enable-tests"]
+  [make "test"]
+]
+install: [make "install"]
+remove: [
+  ["ocamlfind" "remove" "pci"]
+  ["ocamlfind" "remove" "pci_bindings"]
+]
+depends: [
+  "ctypes" {>= "0.4"}
+  "ocamlfind" {build}
+  "ounit" {test}
+  "ocamlbuild" {build}
+]
+available: [ocaml-version >= "4.01.0" & os = "linux"]
+depexts: [
+  [["debian"] ["hwdata" "libpci-dev"]]
+  [["ubuntu"] ["hwdata" "libpci-dev"]]
+  [["centos"] ["hwdata" "pciutils-devel"]]
```

Comment:
Can you please add
```
  [["alpine"] ["hwdata" "pciutils-dev"]]
```
to make it compatible also with alpine?

Reply:
Added in 13834dc